### ISSUE TITLE
Example using `KernelShap` with distributed options

### ISSUE THREAD

**Issue by jklaise** [SeldonIO/alibi#450]
With `ray` we can optionally run `KernelShap` explanations in parallel as documented [here](https://docs.seldon.io/projects/alibi/en/latest/methods/KernelSHAP.html#Running-batches-of-explanations-in-parallel), but it would be nice to have an example and adding some benchmarks (we already have some internally from previous work by @alexcoca).

This is slightly related to #357 but the example should only consist of models that can be serialized by `ray`.

**Comment by alexcoca** [comment]
The DistributedKernelShap repo has some benchmark results one can refer to in the meantime!

**Comment by jklaise** [comment]
For reference, the repo in question: https://github.com/SeldonIO/DistributedKernelShap
